mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -538,6 +517,21 @@ static GlobalStoreOp traceToGlobalStore(Value inp) {
   return allValidUses ? result : GlobalStoreOp();
 }
 
+static AffineMap findAliasingOutputIdxMap(linalg::GenericOp laGeneric,
```

Comment:
I removed this one as I was building up to support transforming store coordinates in the next PR which might be something we dont want to do for variety of reasons. Since we have the check, we can just use the ouput idx map.